README.md

```diff
@@ -1,2 +1,34 @@
 # docker-spark-gpg
-Docker Alpine image with Spark, Python, Java and gpg. Used for local dev environment.
+Docker Alpine image with Spark, Python, Java and gpg. Used for loacal dev environment.
+
+Instructions
+
+
+1. Run the command in order to mount the export-data folder , the TestData folder and the gpg folder on to the root of the Docker container
+
+'docker run -v /Users/yasserkhan/Desktop/docker-spark-gpg/export-data:/root/export-data -v /Users/yasserkhan/Desktop/docker-spark-gpg/Spark:/root/Spark -v /Users/yasserkhan/Desktop/docker-spark-gpg/gpg:/root/gpg -it docker-spark-gpg :latest bash'
```

Comment:
this is too tightly coupled to you local directory structure. The following would be better:
'docker run -v ~/export-data:/root/export-data -v ~/Spark:/root/Spark -v ~/gpg:/root/gpg -it danhill2802/spark-gpg :latest bash'
Also note, the container name isn't `docker-spark-gpg`